src/components/AddItems.js

```diff
@@ -136,7 +136,7 @@ const AddItemsToList = () => {
                     value="7"
                     checked={daysLeftForNextPurchase === 7}
                     onChange={daysLeftForNextPurchaseHandler}
-                    className="mr-2 h-5 w-5 mb-1 text-green-600 focus:ring-1 focus:ring-green-600 cursor-pointer"
+                    className="mb-1 mr-2 h-5 w-5 text-green-600 focus:ring-1 focus:ring-green-600 cursor-pointer"
```

Comment:
The middle and last ones seem perfect now, in both directions. But something is still wrong with the first one, and I can't really figure out why. When looking at dev tools, the "height" is 25.xx while the height of the others is 24 straight. 😕